### DISCUSSION TITLE
Move text to another file like Workbench plugin do but allows typing different context

### DISCUSSION THREAD

**Discussion by Quorafind** [SilentVoid13/Templater#180] [Ideas & Feature Requests]
Workbench plugin works good when I wanted to add same context to text and moved them to same file. But it did't works well when I need to change context in some time.

**Comment by GitMurf** [comment]
@Quorafind did you see the templater script I created last week? Here: [Move/Copy the active line OR selected text to another file - #171](https://github.com/SilentVoid13/Templater/discussions/171)
